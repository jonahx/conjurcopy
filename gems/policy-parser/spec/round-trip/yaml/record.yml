- !group &developers developers

- !user
  id: alice
  annotations:
    hair-color: blonde
