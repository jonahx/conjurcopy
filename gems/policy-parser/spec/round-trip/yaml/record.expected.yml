---
- !group
  id: developers
- !user
  annotations:
    hair-color: blonde
  id: alice
